README.md

```diff
@@ -2,3 +2,10 @@
 
 This is an app to send bulk messages to multiple contacts at same time.<br><br>
 This app is built on flutter using <strong>sms</strong> package but it can only be used on an android device as the package used to send message is not supported by iOS.
+
+### Screenshots
+
```

Comment:
You can also use `markdown` syntax.

`![sreeenshot](https://github.com/vanshg395/sms_app/blob/master/Screenshot_1568227014.png?raw=true)`

![sreeenshot](https://github.com/vanshg395/sms_app/blob/master/Screenshot_1568227014.png?raw=true)